packages/connect/src/ws_orderbook_channel.ts

```diff
@@ -80,13 +94,21 @@ export class WebSocketOrderbookChannel implements OrderbookChannel {
         if (!_.isUndefined(this._connectionIfExists)) {
             this._connectionIfExists.close();
         }
+        if (!_.isUndefined(this._heartbeatTimerIfExists)) {
+            clearInterval(this._heartbeatTimerIfExists);
+        }
     }
     private _getConnection(callback: (error?: Error, connection?: WebSocket.connection) => void) {
         if (!_.isUndefined(this._connectionIfExists) && this._connectionIfExists.connected) {
             callback(undefined, this._connectionIfExists);
         } else {
             this._client.on(WebsocketClientEventType.Connect, connection => {
                 this._connectionIfExists = connection;
+                if (this._heartbeatIntervalMs !== 0) {
```

Comment:
This should now check:

```
if (this._heartbeatIntervalMs >= MIN_HEARTBEAT_INTERVAL) {
```

And if not, it should throw an error since it means we did not properly validate the user supplied value (i.e developer error).